src/surforama/app.py

```diff
@@ -163,11 +167,9 @@ def _get_valid_image_layers(self, combo_box) -> List[Image]:
         ]
 
     def get_point_colors(self, points):
-        point_indices = points.astype(int)
-
-        point_values = self.volume[
-            point_indices[:, 0], point_indices[:, 1], point_indices[:, 2]
-        ]
+        point_values = map_coordinates(
+            self.volume, points.T, order=1, mode="nearest"
```

Comment:
How is the speed when doing linear interpolation? Does it still feel responsive? Do you think this should be an option? In any case, we can make that option later.

Reply:
For the demo example on my local computer, it still ran very smooth with the demo examples (higher order interpolations were very slow though). But yes, I guess for larger meshes, this can become problematic.
Would be good to have that as an option.